schema/schema.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE item (
+    id  INT GENERATED ALWAYS AS IDENTITY,
+    genre_id INT,
+    author_id INT,
+    label_id INT,
+    publish_date DATE,
+    archived BOOLEAN,
+    PRIMARY KEY(id),
+    FOREIGN KEY (genre_id) REFERENCES genres (id),
+    FOREIGN KEY (author_id) REFERENCES authors (id),
+    FOREIGN KEY (label_id) REFERENCES labels (id)
+);
+
+CREATE TABLE music_albums (
+    id  INT,
+    name VARCHAR(100),
+    on_spotify BOOLEAN,
+    PRIMARY KEY(id),
+    FOREIGN KEY(id) REFERENCES item(id)
```

Comment:
Remember, in SQL doesn't exist inheritance, so `music_albums` table should include all `item` columns. Please, move all the columns from `item` table to `music_albums` and delete `item` table